src/main/webapp/user_profile_style.css

```diff
@@ -294,3 +294,29 @@ table button:hover i {
   margin: 2px;
   color: red;
 }
+
+#message-container {
+  max-height: 20%;
+  overflow-y: auto;
+  border: 4px solid black;
+  border-radius: 2px;
+  margin-bottom: 5px;
+}
+
+.sentByOthers {
+  border: 2px solid #dedede;
+  background-color: #f1f1f1;
+  border-radius: 5px;
+  padding: 10px;
+  margin: 10px 10px;
+}
+
+.sentByMe {
+  color: white;
+  background-color: #1aff1a;
```

Comment:
nit: could we use a less bright green for this? :)

Reply:
Sure. I will change to a darker green in the new commit.